SoftLayer/CLI/block/object_storage_permission.py

```diff
@@ -0,0 +1,44 @@
+"""Display details for a specified cloud object storage."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+import SoftLayer
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+
+
+@click.command(cls=SoftLayer.CLI.command.SLCommand, )
+@click.argument('object_id')
+@environment.pass_env
+def cli(env, object_id):
+    """Display details for a cloud object storage."""
+
+    block_manager = SoftLayer.BlockStorageManager(env.client)
+
+    cloud = block_manager.get_network_message_delivery_accounts(object_id)
+    end_points = block_manager.get_end_points(object_id)
+
+    table = formatting.Table(['name', 'value'])
+
+    table_credentials = formatting.Table(['Id', 'Username', 'Password', 'Description'])
+
+    for credential in cloud.get('credentials'):
+        table_credentials.add_row([credential.get('id'),
+                                   credential.get('username'),
+                                   credential.get('password'),
+                                   credential['type']['description']])
+
+    table_url = formatting.Table(['region',
+                                  'location',
+                                  'type',
+                                  'url'])
+    for end_point in end_points:
+        table_url.add_row([end_point.get('region') or '',
+                           end_point.get('location') or '',
+                           end_point.get('type'),
+                           end_point.get('url'), ])
+
+    table.add_row(['UUID', cloud.get('uuid')])
+    table.add_row(['Credemtial', table_credentials])
```

Comment:
Replace `Credemtial` by `Credential`, and maybe like object storage could have many credentials, it is better to use the plural form